bin/init

```diff
@@ -6,6 +6,14 @@ set -e
 
 : ${VELUM_PORT:=80}
 
+setup_root_ca() {
+  # Velum is going to need this CA to talk to the running CaaSP Cluster
+  [ -f "/etc/pki/trust/anchors/SUSE_CaaSP_CA.pem" ] && return
```

Comment:
I think we can tweak this in case the CA changes for whatever reason in the future (I can think of a process in which the administrator completely requests a CA reissuing along with all certificates because of a security issue within the company for example). Maybe we can only return if `/etc/pki/ca.crt` sha2 does not match `/etc/pki/trust/anchors/SUSE_CaaSP_CA.pem` sha2. In any case I'm fine as it is right now, I'd create an issue for not forgetting that at least.

Reply:
oh, good point.